.gitignore

```diff
@@ -4,3 +4,5 @@ package-lock.json
 dist/main.js
 webpack.config.js
 main.js
+dist/main.*.js
```

Comment:
the line below should cover this one as well, maybe keep the more general rule instead of both?